src/index.js

```diff
@@ -107,6 +108,12 @@ module.exports = (
   else if (typeof externals === 'object')
     Object.keys(externals).forEach(external => externalMap.set(external, externals[external]));
 
+  const aliasMap = {};
+  if (Array.isArray(aliasMap))
+    alias.forEach(alias => aliasMap[alias[0]] = alias[1]);
+  else if (typeof alias === 'object')
+    Object.keys(alias).forEach(from => aliasMap[from] = alias[from]);
```

Comment:
```suggestion
    aliasMap = alias;
```

I think you can avoid iteration here an assign instead